templates/common/_base/units/ovsdb-server.service.yaml

```diff
@@ -5,3 +5,6 @@ dropins:
     contents: |
       [Service]
       Restart=always
+      ExecStartPre=-/bin/sh -c '/usr/bin/chown -R ${OVS_USER_ID} /var/lib/openvswitch'
```

Comment:
Why not use `:$${OVS_USER_ID##}`?

Reply:
It was insufficient in my case - not entirely sure why. This script should ensure that a different user owns these folders anyway (at least its the case in current OKD 4.6 -> 4.7 upgrade)

Reply:
It's unclear to me what insufficient means. What happens if you do `:$${OVS_USER_ID##*:}`?

Reply:
OVS_USER_ID is coming from openvswitch.useropts which in itself is getting set from either default.conf or /etc/config/openvswitch.

Reply:
>It's unclear to me what insufficient means. What happens if you do :$${OVS_USER_ID##*:}?

`ovsdb-server` service fails on boot. I can't ssh and check more details.

Changing this to `chown -R ${OVS_USER_ID}` makes the machine boot